Cargo.toml

```diff
@@ -8,6 +8,7 @@ members = [
   "helix-dap",
   "helix-loader",
   "helix-vcs",
+  "helix-parsec",
```

Comment:
*nit* name parsec is not very obvious to me by just reading it.

Reply:
I like it because it matches to a whole bunch of other parsing libraries https://en.wikipedia.org/wiki/Parsec_(parser)